### ISSUE TITLE
postgresql 10.4.1 start error 

### ISSUE THREAD

**Issue by jerryliuxingrui** [timescale/timescaledb#1104]
**Relevant system information:**
 - OS: [Windows 2008 R2 x64, etc]
 - PostgreSQL version (output of `postgres --version`): [10.4.1]
 - TimescaleDB version (output of `\dx` in `psql`): [1.2.1]
 - Installation method: [ "apt install"]

**Describe the bug**
 HKT [616] :  can't load library "D:/Program Files/PostgreSQL/10/lib/timescaledb.dll": Access is denied.

**To Reproduce**

**Expected behavior**
A clear and concise description of what you _expected_ to happen.

**Actual behavior**
A clear and concise description of what _actually_ happened.

**Screenshots**

**Additional context**

**Comment by RobAtticus** [comment]
You say you used `apt install` with a Windows system -- is this through the Windows Linux Subsystem? We have not tested using WLS so I wouldn't be confident it would work.

**Comment by jerryliuxingrui** [comment]
> 您说您使用`apt install`的是Windows系统 - 这是通过Windows Linux子系统吗？我们没有使用WLS进行测试，因此我不相信它会起作用。

I'm sorry,I didn't describe it clearly. I installed it with setup.exe, the installation file of windows. The same way works on another Windows 2008 R2, but one server can't start the PostgreSQL service. In the Windows Event Viewer, I found the following error: HKT [616]: can't load library "D:/Program Files/PostgreSQL/10/lib/timescaledb.dl" Access is denied.

**Comment by RobAtticus** [comment]
Does the file exist at that location? 

Is this during installation or restarting the server? The installer will fail if the server is running with timescaledb already installed since it cannot overwrite a file that is in use.

**Comment by mfreed** [comment]
Hi @jerryliuxingrui Just following up to see if you can provide that additional information.

**Comment by davarihn** [comment]
Hi @jerryliuxingrui , @mfreed @RobAtticus 

I have the same problem when I try to install Timescaledb. 

- OS: Windows 10
- PostgreSQL: 12.4
- Timescaledb: 1.7.4
- Installation method: setup.exe

This is the error I'm getting:

ERROR: could not copy file 'timescaledb-1.7.4.dll': open C:/PROGRA~1/POSTGR~1/12/lib/timescaledb-1.7.4.dll: Access is denied.

![1](https://user-images.githubusercontent.com/45049016/93345806-dd998f80-f7f8-11ea-900b-292872602707.PNG)

This file exists in the timescaledb installation folder. 

I've checked/tried the following:

Have Visual C++ 2015-2019 redistributable X64 - 14.24.28127 installed.
Added the Postgres installation folder to windows PATH.
Checked the permissions for the dll file.
Copied the dll file manually to the destination folder (Program Files/PostgreSQL/12/lib).
Removed and reinstalled PostgreSQL and Timescaledb. 

I still get the same error.
